app/javascript/src/BackendApis/components/BackendSelect.jsx

```diff
@@ -35,14 +36,15 @@ const BackendSelect = ({ backend, backends, onSelect, onCreateNewBackend, error
         // $FlowIssue[incompatible-type] backend is compatible with null
         item={backend}
         items={backends.map(b => ({ ...b, description: b.privateEndpoint }))}
+        itemsCount={backends.length}
         cells={cells}
-        helperTextInvalid={error}
-        isValid={!error}
-        modalTitle="Select a Backend"
         // $FlowIssue[incompatible-type] It should not complain since Record.id has union "number | string"
         onSelect={onSelect}
         header="Most recently created Backends"
-        footer="View all Backends"
+        title="Select a Backend"
+        placeholder="Select a Backend"
+        footerLabel="View all Backends"
+        helperTextInvalid={error}
```

Comment:
Is `fetchItems` missing?
In [SelectWithModal](https://github.com/3scale/porta/pull/2610/files#diff-92a9d2732a616fdc9076e8a68a9508f1a948431a64bf7f535bc0221d911c7c4cR32) is not optional.

Reply:
Yes it's intentional, `SelectWithModal` is supposed to work with remote data always. I'll update them in follow up PRs like the one for New Application page.